src/api-service/__app__/onefuzzlib/events.py

```diff
@@ -33,9 +33,53 @@ def get_events() -> Optional[str]:
         return None
 
 
+def log_event(event: Event, event_type: EventType) -> None:
+    
+    clone_event = event.copy(deep=True)
+    log_event_recurs(clone_event)
```

Comment:
Just curious if this recursive call would ever cause stack overflow if it reaches max recursion depth?

Reply:
Python's recursion limit is 1000 deep.  The `Event` types shouldn't have that much nested data such that we hit this in practice.